src/extensions/default/InlineColorEditor/ColorEditor.js

```diff
@@ -42,6 +42,34 @@ define(function (require, exports, module) {
     var STEP_MULTIPLIER = 5;
 
     /**
+     * Convert 0x notation into hex6 format for tinycolor
+     * compatibility: ("0xFFAACC" => "#FFFFFF")
+     */
+    function color0xToHex(color,toStr){
+        var format0xToHexColor = color.replace("0x","#");
+        var hexColor = tinycolor(format0xToHexColor);
+        hexColor._format = "0x";
+
+        if(toStr !== null){
+            hexColor.toString();
+        }
+        return hexColor;
+    }
+
+    function as0xString(color){
+        return color.toHexString().replace("#","0x");
+    }
+
+    function checkSetFormat(color,toStr){
+        if((/^0x/).test(color)){
+            var colorRes = color0xToHex(color,toStr);
```

Comment:
do not create a variable if it's used only once, you can do `return color0xToHex(color,toStr)` directly